projects/npm-tools/packages/npm-scripts/src/utils/createEsm2AmdExportsBridges.js

```diff
@@ -71,18 +91,34 @@ function createEsm2AmdExportsBridges(projectDir, buildConfig, manifest) {
 		// Also, depending for npm-scoped scoped packages, and additional folder
 		// level appears under `/o/js/resolved-module/...`.
 		//
+		// Also, for internal module paths, we must add more `../`s.
+		//
+
+		let rootDir = '../..';
+
+		if (exportItem.name.startsWith('@')) {
+			rootDir += '/..';
+		}
+
+		const hopsToAdd = modulePath.split('/').length - 1;
```

Comment:
is `hops` a technical thing? It totally makes sense to me, but wasn't sure if it was actually a thing

Reply:
Not really in this context, but I couldn't come up with a better word 🤷.

In computer networking terminology [a hop is a jump from one router to another](https://www.lifewire.com/what-are-hops-hop-counts-2625905). It resembled this case (for me).